airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl.tsx

```diff
@@ -219,6 +221,14 @@ const ConnectorServiceTypeControl: React.FC<{
     [setValue, onChangeServiceType]
   );
 
+  const onMenuOpen = () => {
+    const eventName =
+      formType === "source"
+        ? "Airbyte.UI.NewSource.SelectionOpened"
```

Comment:
Shouldn't we have separate type/enum or const for event names if we are going to follow some common principle?

Reply:
I would refrain for now from using an enum, since we're trying to flatten out those events, i.e. the same ID should only be used in one place, thus I don't see a huge benefit in having a reusable enum. If I'd rather suggest having a utility method generating those (to ensure they follow some common principal). Though I currently feel this doesn't actually improve readability (and maintenance) of that code, thus would leave it out for now, and we can revisit this, if we feel this is actually causing issues.